manifests/4.5/local-storage-operator.v4.5.0.clusterserviceversion.yaml

```diff
@@ -219,6 +219,10 @@ spec:
                         valueFrom:
                           fieldRef:
                             fieldPath: metadata.namespace
+                      - name: POD_NAME
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.name
```

Comment:
Does this CSV requires same RBAC change that you did in `deploy/rbac.yaml` file?

Reply:
yes